src/classes/STG_InstallScript.cls

```diff
@@ -66,7 +69,9 @@ global without sharing class STG_InstallScript implements InstallHandler {
 
             disablePreferredEmailEnforcement(previousVersion);
 
-            disablePreferredPhoneEnforcement(previousVersion);            
+            disablePreferredPhoneEnforcement(previousVersion);
+
+            disableAutoPopulationOfAffilOnStudentCourseCxn(previousVersion);
```

Comment:
just wanting to verify that this is indeed doing the right thing and we won't accidentally turn it off for a customer who gets this after release, but then gets an upgrade in the future. It looks like we are comparing it to 1.92 in this method, so it will only turn it off if their previous version is less than 1.92, is that correct? Sorry if this is obvous, my brain is foggy today.

Reply:
@lifewithryan , yep I believe so.